src/App.js

```diff
@@ -1,23 +1,43 @@
 import React, { useState } from 'react';
+import { HotKeys } from 'react-hotkeys';
 
 import Background from './Background.svg';
 import Controls from './UI/Controls';
-import Vectorize from './Vectorize';
+import Glasses from './Glasses';
 import HighScores from './UI/HighScores';
 import PoseNet from './PoseNet';
 import PoseName from './UI/PoseName';
 import PoseScore from './UI/PoseScore';
+import Vectorize from './Vectorize';
 import { Tadasana } from './Yoga/Pose/Tadasana';
 import { VirabhadrasanaII } from './Yoga/Pose/VirabhadrasanaII';
 import Yoga from './Yoga';
 
 import 'bootstrap/dist/css/bootstrap.min.css';
 
+const _keyMap = {
+  GLASSES_MODE: ['command+alt+g'],
+  RESET_MODE: ['command+alt+r'],
+  VECTORS_MODE: ['command+alt+v'],
+};
+
 function App() {
   const _height = 500;
   const _width = 500;
+
   const [pose, setPose] = useState(null);
   const [maxScore, setScore] = useState(0);
+  const [glasses, setGlasses] = useState(false);
+  const [vectors, setVectors] = useState(false);
+
+  const handleKeyPress = {
+    GLASSES_MODE: () => setGlasses(true),
+    RESET_MODE: () => {
+      setGlasses(false);
```

Comment:
should `RESET_MODE` also reset the `maxScore` ?

Reply:
Oh good question... maybe I should have named this differently? It is a handler to turn off the glasses and vectorize HUD. What do you think?

Reply:
I think that's cool, we do have a button to reset the score after all.